posthog/api/test/test_event_definition.py

```diff
@@ -234,6 +234,36 @@ def test_event_type_event_posthog(self):
         assert response.json()["count"] == 1
         assert response.json()["results"][0]["name"] == "$pageview"
 
+    @patch("posthog.models.Organization.is_feature_available", return_value=False)
+    def test_update_event_definition_without_taxonomy_entitlement(self, mock_is_feature_available):
+        event_definition = EventDefinition.objects.create(team=self.demo_team, name="test_event")
+
+        response = self.client.patch(
+            f"/api/projects/@current/event_definitions/{event_definition.id}",
+            {"name": "updated_event"},
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        event_definition.refresh_from_db()
+        self.assertEqual(event_definition.name, "updated_event")
+        self.assertEqual(response.json()["name"], "updated_event")
+
+    @patch("posthog.settings.EE_AVAILABLE", True)
+    @patch("posthog.models.Organization.is_feature_available", return_value=True)
+    def test_update_event_definition_with_taxonomy_entitlement(self, *mocks):
+        event_definition = EventDefinition.objects.create(team=self.demo_team, name="test_event")
+
+        response = self.client.patch(
+            f"/api/projects/@current/event_definitions/{event_definition.id}",
+            {"verified": True},  # verified field only exists in enterprise serializer
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        # Verify the enterprise-only field was updated
+        self.assertEqual(response.json()["verified"], True)
```

Comment:
we tend to use `assert x == y` now... i think it's recommended by pytest and we get much nicer output on failure

(i update these as i come across them)

also...

should we have, if we don't already, a test that asserts you can't set verified without EE available?